decode.go

```diff
@@ -253,3 +253,8 @@ func NewDecoder(r io.ReadCloser) (*Decoder, error) {
 func Decode(r io.ReadCloser) (*Decoder, error) {
 	return NewDecoder(r)
 }
+
+// Decode decodes all mp3 data from d into dst - memory usage can be large, use with caution
+func (d *Decoder) Decode(dst io.Writer) (int64, error) {
+	return io.Copy(dst, d)
```

Comment:
In Go, destination is always first :-)

As I'd like keep API as minimum as possible, I'd not to add this.